content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -52,24 +65,33 @@ ECS tasks that run on Fargate are already fully allocated in the CUR. CCM enrich
 
 ### Everything else
 
-Any cost other than EC2, computed for instances hosting Kubernetes pods or ECS tasks, is given the same value and tags as the source metric, `aws.cost.amortized`.
+Cost other than EC2/EBS, computed for instances hosting Kubernetes pods or ECS tasks, is given the same value and tags as the source metric, `aws.cost.amortized`.
```

Comment:
```suggestion
All other costs (costs that are not related to EC2 or EBS, and are not computed for instances hosting Kubernetes pods or ECS tasks) are given the same value and tags as the source metric, `aws.cost.amortized`.
```

@kaankoseler I'm trying to reword this sentence to make it more clear. Can you please check that my change doesn't change the intended meaning of the sentence?

Reply:
Hi @cswatt thank you for the review.

This is a tough sentence to work with, I think. The revision does seem to lose the meaning somewhat.  
The idea that we are trying to convey is that this subset of costs is what's being targeted with the feature:
`EC2/EBS costs for EC2 instances hosting Kubernetes pods or ECS tasks`


Any cost that does not fall under that umbrella is considered `Everything else` and left as is with the source metric. 
So we somehow need to keep that sentence above as one coherent whole, IMO.

Reply:
As the list of allocated cost types is growing, this sentence might just not be scalable 😆 I wonder if we even need to re-list all the costs above. One option is just drop the explanation:
"All other costs are given the same value and tags as the source metric, `aws.cost.amortized`."

Or, if we want an explanation, to split it up into multiple sentences:
"All other costs are given the same value and tags as the source metric, `aws.cost.amortized`. This includes costs for all other AWS services, as well as EC2 and EBS costs not associated with container orchestration."

Reply:
I would prefer the former, personally. It should be pretty obvious from the sections and the preceding text of the document

Reply:
@cswatt what do you think about just keeping this as:
`"All other costs are given the same value and tags as the source metric, aws.cost.amortized."`

Reply:
How about "The information on this page pertains to EC2/EBS costs for EC2 instances hosting Kubernetes pods or ECS tasks. All other costs are given the same value and tags as the source metric, `aws.cost.amortized`." —does that work?

Reply:
Yes, that works! cc @enemyghost

Reply:
I talked to @enemyghost offline, can we slightly modify to say `... pertains to EC2 Compute and EBS costs...`?